src/api/app/models/notifications/rss_feed_item.rb

```diff
@@ -20,6 +20,18 @@ def self.cleanup
       end
     end
   end
+
+  def title
+    event.subject
+  end
+
+  def description
+    ApplicationController.renderer.new.render(
```

Comment:
@hennevogel I agree that this rendering shouldn't be done in the model. Seems like the Mailer class would be a better place for it?